CONTRIBUTING.md

```diff
@@ -0,0 +1,65 @@
+# CONTRIBUTING GUIDELINES
+
+NovelRT is a community-driven project. Contribution is always welcome, and much appreciated.
+It is also how it'll be developed.
+
+First, please take a moment to review our [code of conduct](CODE_OF_CONDUCT.md). We expect that all project participants will adhere to it, and any actions taken that go against the Code of Conduct will not be tolerated.
+
+These guidelines are an attempt at better addressing the incoming issues and pull requests. 
+Please read them closely, and feel free to ask any questions if needed.
```

Comment:
@capnkenny gonna want you to check my suggestion here, I'm not 100% sure on my suggestion here but I do think the original could be improved.

```suggestion
These guidelines are an outline of the contribution process. Please read them closely, and feel free to ask any questions if needed.
```

Reply:
I think it summarizes the phrasing well here @RubyNova